data/ideas.json

```
@@ -1,4 +1,10 @@
 [
+    {
+        "title": "File Protection System",
+        "description": "This is a small File Protection System Made using only Java.",
+        "level": "intermediate",
+        "tags": "java, swing, security, file, protection, project"
```

The first tag is used as a unique key to match the idea and project, and remove the technology or library because it could be programmed by another language.
```suggestion
        "tags": "file-protection-system, security"
```